hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/Rf part.png">
+</p>
+
+<div align="center">
+
+<b>Figure 2. RF part</b> 
+
+</div>
+
+
+
+### 2.2.1. Layout of the matching network   
+
+The layout design of the match is critical to achieve the targeted power, the optimum sensitivity and efficiency. 
+This is especially true for the RX path, which can be easily detuned by the PCB parasitics.   
+ 
+
+Although the size of a matching network is relatively small, due to the high frequency the matching is sensitive to the physical parameters of the matching layout (beside the component values, of course).  
+Compared to a reference layout design:
+> - Different component distances and trace lengths introduce different parasitic inductances
+> - Different substrate thickness, dielectric constant and gaps between traces introduce different parasitic capacitances
+> - Different component distance/orientation/relative orientation introduce different coupling between components
+> - Different component types/sizes introduce different component parasitics
+
+These can cause detuning of the reference matching or filtering and/or detuning of the load of the RF crystal. The possible effects can be:
+> - Decreased TX output power of the fundamental
+> - Decreased RX sensitivity
+> - Increased spurious emission levels  
+> - Increased current consumption
+> - Frequency offset between different boards
+
+
+
+# 3. Layout Design Practices 
+## 3.1. Layout Design Practices - 2.4 GHz matching network  
+
+
+1. Minimize distance between neighboring components. Matching network components should be close to each other and to the RF IC. For nearby components use the width of the pads for trace width
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/1.png">
+</p>
+
+<div align="center">
+
+<b>Figure 3.</b> 
+
+</div>
+
+
+
+
+2. The first component of the matching network should be placed as close to the 2G4RF_IOP pin of the RF IC as possible to reduce the series parasitic inductance and to avoid any detuning effects.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/2.png">
+</p>
+
+<div align="center">
+
+<b>Figure 4.</b> 
+
+</div>
+
+
+3. Couplings through the ground pads can occur between the nearby filtering capacitors (especially at high harmonics). This can decrease the effectiveness of the low-pass filtering and can cause higher harmonics that can be observed both conducted and radiated.
+ To avoid the possible high harmanic levels, it is recommended to connect the nearby filtering capacitors to ground planes on different sides of the transmission line.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/3.png">
+</p>
+
+<div align="center">
+
+<b>Figure 5.</b> 
+
+</div>
+
+**Note:** It is true only in the case of Series 1 chips. In the case of EFRXG2X Series 2  chips the capacitors should be connected on same sides of the transmissin line. 
```

Comment:
Change "Series 1 chips" to "EFR32xG1x Series 1 chips".
Change "EFRXG2X Series 2  chips" to "EFR32xG2x Series 2 chips".
Change "transmissin" to "transmission".